ci/README.md

```diff
@@ -0,0 +1,110 @@
+# Continuous Integration Setup
+
+The CI environment is maintained by the SAP Business Technology Platform on Cloud Foundry team for Routing and Networking (CFN).
```

Comment:
From my POV, team name is not needed, we can replace "CFN vault" with "team's vault" below